.rubocop.yml

```diff
@@ -204,3 +207,62 @@ Layout/FirstMethodArgumentLineBreak:
 # Opt-in
 Layout/FirstMethodParameterLineBreak:
   Enabled: true
+
+# refute and assert are nice opposites
+Rails/RefuteMethods:
+  Enabled: false
+
+# we enforce UTC for Time.now, so no harm done
+Rails/TimeZone:
+  Enabled: false
+
+Rails/Date:
+  Enabled: false
+
+# we use this a lot in tests/db and sometimes want to skip side-effects
+Rails/SkipsModelValidations:
+  Enabled: false
+
+# works the same, so no harm done and sometimes we override the methods
+Rails/DynamicFindBy:
+  Enabled: false
+
+# we inherit a lot, so it complains about missing actions
+Rails/LexicallyScopedActionFilter:
+  Enabled: false
+
+# we don't use ApplicationRecord
+Rails/ApplicationRecord:
+  Enabled: false
+
+# TODO: enable and fix
+Rails/ActiveRecordAliases:
+  Enabled: false
+
+# Not helpful
+Rails/OutputSafety:
+  Enabled: false
+
+# Not helpful
+Rails/ReversibleMigration:
+  Enabled: false
+
+# Not helpful
+Rails/Output:
+  Enabled: false
+
+# Broken see https://github.com/rubocop-hq/rubocop-rails/issues/147
+Rails/HasManyOrHasOneDependent:
+  Enabled: false
+
+# TODO: enable and fix
+Rails/InverseOf:
+  Enabled: false
```

Comment:
can this replace some of the cleanliness tests?

Reply:
yeah pretty sure it will ... but it's complaining about a few so I want to dig in a bit deeper